htmd/builder/builder.py

```diff
@@ -243,6 +243,12 @@ def _checkMixedSegment(mol):
                        'Please assign separate segments to them or the build procedure might fail.'.format(intersection))
 
 
+def _checkLongResnames(mol, aliasresidues):
```

Comment:
Is this in the scope of this PR? It seems like this PR is mixing too many things.

Reply:
I know but I'm really too lazy to undo commits and make a new PR.

Reply:
Well, not much I can do then :)

Reply:
I don't get this. All of this is in master already. Not on this branch...